mne/realtime/epochs.py

```diff
@@ -314,6 +314,7 @@ def _process_raw_buffer(self, raw_buffer):
         verbose = 'ERROR'
         sfreq = self.info['sfreq']
         n_samp = len(self._raw_times)
+        prev_samp = int(self._find_events_kwargs['min_samples'] - 1)
```

Comment:
hmm ... indeed. It would be nice to have a fix for that as well along with a test. I don't mind splitting it into two PRs if you want. But since you are going to use it anyway, it is perhaps better to squeeze it into the same PR?

I am presuming that such a duplicate event would be in `self._event_backlog` ... unless your `min_samples` is larger than epoch length (which is unlikely? or you could add a condition to check it). Now, you could do an `np.unique` somewhere here I suppose: https://github.com/mne-tools/mne-python/blob/master/mne/realtime/epochs.py#L333 ? So, you could make `prev_sample` large so that you take the entire last buffer. And then take the unique events among all the events found in this buffer from after the last event in event_backlog.

ok, maybe the logic sounds a big complicated when I say it this way. I would recommend that you write the test first so that it fails. And then write the code to make it pass.